lifelines/plotting.py

```diff
@@ -1,15 +1,12 @@
 # -*- coding: utf-8 -*-
 from __future__ import print_function
 
-# plotting
 import numpy as np
-import matplotlib as mpl
-from matplotlib import pyplot as plt
-
 from lifelines.utils import coalesce
 
 
 def is_latex_enabled():
+    import matplotlib as mpl
```

Comment:
Docstrings have to be the *first* thing in the method, no?
So it should be moved to after the docstring.